project_national/fact_sheets_category_3.yml

```diff
@@ -0,0 +1,354 @@
+                         # Another option: $1064. $760 for replacing electric dryer in REMDB, then apply 1.4 inflation factor (Jan 2010 to Apr 2023)
+            multiplier: Fixed (1)
+        lifetime: 13 # 13 is from REMDB https://remdb.nrel.gov/measures?group_id=4&component_type_id=374&actionId=1925&bcId=5849
+                     # 18 from IEA https://www.eia.gov/analysis/studies/buildings/equipcosts/pdf/appendix-a.pdf
+
+  - upgrade_name: Induction Range & Oven for Non-electric Cooking
+    options:
+      - &induction_range_oven_for_non-electric_cooking 
+        option: Cooking Range|Electric Induction
+        apply_logic:
+          - or:
+            - Cooking Range|Gas
+            - Cooking Range|Propane
+        costs:
+          - value: 1628.53 #($1399*1.07 sales tax+ 94*1.4 labor) https://www.ikea.com/us/en/p/tvaersaeker-range-with-induction-cooktop-stainless-steel-40466006/
+            multiplier: Fixed (1)
+        lifetime: 13 # from REMDB https://remdb.nrel.gov/measures?group_id=4&component_type_id=279&actionId=2204&bcId=6961
+                     # 17 from IEA https://www.eia.gov/analysis/studies/buildings/equipcosts/pdf/appendix-a.pdf
+
+  - upgrade_name: Conventional Range & Oven for Non-electric Cooking
+    options:
+      - &conventional_range_oven_for_non-electric_cooking 
+        option: Cooking Range|Electric Resistance
+        apply_logic:
+          - or:
+            - Cooking Range|Gas
+            - Cooking Range|Propane
+        costs:
+          - value: 1830.2 #price1: $674.99-2499.99 for Convection Range in BestBuy
+                         #price2: $799- for Convection Range in Home Depot. The upper end of the range is missing because the search engine of home depot for electric convection range are induction range with convection oven. kind of difficult to find the highest price for convection range
+                         # 1830.2=1587.49*1.07+94*1.4, price*1.07 sales tax + 94*1.4 labor.
+                         # Anopther option: $1288. $920 for replace non electric range to electric range in REMDB, then apply 1.4 inflation factor (Jan 2010 to Apr 2023)
+            multiplier: Fixed (1)
+        lifetime: 13 # from REMDB https://remdb.nrel.gov/measures?group_id=4&component_type_id=279&actionId=2204&bcId=6961
+                     # 17 from IEA https://www.eia.gov/analysis/studies/buildings/equipcosts/pdf/appendix-a.pdf
+
+
+  - upgrade_name: Electric Pool Heaters
+    options:
+      - &electric_pool_heaters 
+        option: Misc Pool Heater|Electricity
+        apply_logic:
+          - Misc Pool Heater|Natural Gas
+
+  - upgrade_name: Electric Spa Heaters
+    options:
+      - &electric_spa_heaters 
+        option: Misc Hot Tub Spa|Electricity
+        apply_logic:
+          - Misc Hot Tub Spa|Natural Gas
+
+  - upgrade_name: High-Efficacy LED Lighting
+    options:
+      - &lighting_100_percent_led
+        option: Lighting|100% LED
```

Comment:
I don't think our baseline "100% LED" option is "high efficacy" - probably need to check in with @trynthink ?

Reply:
Conclusion from our meeting: we are going to use the standard LEDs that are currently in ResStock. Let's update the documentation to be standard LEDs but leave the YAML as-is.